daemon/api_debug_test.go

```diff
@@ -34,25 +35,27 @@ import (
 var _ = check.Suite(&postDebugSuite{})
 
 type postDebugSuite struct {
-	APIBaseSuite
+	apiBaseSuite
 }
 
 func (s *postDebugSuite) TestPostDebugEnsureStateSoon(c *check.C) {
 	s.daemonWithOverlordMock(c)
 
 	soon := 0
-	ensureStateSoon = func(st *state.State) {
+	var origEnsureStateSoon func(*state.State)
```

Comment:
Since the assignment below uses `:=`, what is the point of declaring this explicitly?